public/index.html

```diff
@@ -10,12 +11,28 @@
 
     <link href="https://fonts.googleapis.com/css?family=Muli:400,600,700" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css?family=Fira+Mono&text=0123456789ABCDEFGHJKLMNPQRSTUVXY" rel="stylesheet">
-  </head>
-  <body>
+    <link href="loading.css" rel="stylesheet">
```

Comment:
I'll go ahead with those changes, but the last one. I made it this way with CSP (disallowing inline styles) in mind. What do you think?

Reply:
With the fonts, it so no easy: https://developers.google.com/web/updates/2016/02/font-display

Reply:
Yeah, ok, I also don't know whether it would actually be beneficial to inline the style. By not inlining, it can at least download stuff in parallel.

Loading the fonts is done by linking to the google stylesheets though and the download of the stylesheets is render blocking even though the download of the font itself is not. Thus, we should move these `<link rel="stylesheet">` tags to the end of `<body>` or maybe just include the content of the google font stylesheet into our stylesheet. We'd need to be careful to include the right fallbacks though, as google serves different stylesheets depending on the user's browser.
So, performance wise, including that stuff into our stylesheet might actually be best.

Reply:
BTW, `<div style="flex-grow: 1;"></div>` will also not work with CSP in place. Also see my other comment for an alternative approach.

Reply:
The latter: I know, already replaced it, just forgot to push.

Reply:
The former: we could use a self-hosted version of webfontloader, or program the functionality ourselves. But I doubt it's worth the effort for 3 seconds without fonts (takes a lot longer to load with slow 3G)

Reply:
Yeah, hosting that ourselves doesn't make much sense.
I was really just talking about copying the content of `https://fonts.googleapis.com/css?family=Muli:400,600,700` and `https://fonts.googleapis.com/css?family=Fira+Mono&text=0123456789ABCDEFGHJKLMNPQRSTUVXY` into our css.
And maybe making sure that links to woff (google hosted) as a fallback are also included.
But probably even woff2 is enough. Browser support is pretty good.